Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -441,8 +450,8 @@ InternalBootPickerEntryDraw (
   //
   // Draw the label horizontally centered.
   //
-  ASSERT (Label->Width  <= BOOT_ENTRY_DIMENSION);
-  ASSERT (Label->Height == BOOT_ENTRY_LABEL_HEIGHT);
+  //ASSERT (Label->Width  <= BOOT_ENTRY_DIMENSION);
+  //ASSERT (Label->Height == BOOT_ENTRY_LABEL_HEIGHT);
```

Comment:
Is this correct?

Reply:
Height: macOS makes height 12, built-in renderer makes height 13. And as we are reading it from file, it may contain any height if future macOS decides to change it.

Reply:
Width: as we don't support scrolling labels now, do we want to be unable to boot in DEBUG because of this assert?

Reply:
I think it should at least be sanitised, but sticking to what Apple has now sounds good to me.